src/main/java/RedBot/Bot.java

```diff
@@ -87,12 +87,20 @@ else if(message[1].equalsIgnoreCase("random")){ //Completely random quote
                     "My reply is no",
                     "My sources say no",
                     "Outlook not so good",
-                    "Very doubtful"
+                    "Very doubtful", 
+                    //shit after here is added by Egg
+                    "Are you sure you want to ask that?",
+                    "Perhaps",
+                    "Sure, why not eh",
+                    "Absolutely not you donkey!",
+                    "Stop asking me dumb questions"
+                    "Wow, that is actually a good question, FINALLY!"
+                    "Tbh, I have no idea."
```

Comment:
Some of these aren't even a clear yes or no answer

Reply:
its 8ball.

Reply:
In any case, I followed the [official 8ball responses](http://www.otcpas.com/advisor-blog/magic-8-ball-messages/), so no need to add more.